src/Tempest/Core/src/Commands/DiscoveryClearCommand.php

```diff
@@ -2,7 +2,7 @@
 
 declare(strict_types=1);
 
-namespace Tempest\Console\Commands;
+namespace Tempest\Core\Commands;
```

Comment:
I moved the discovery commands to Core, similar to how the `InstallCommand` is in core.